rails/benchmarks/activerecord_complex.rb

```diff
@@ -0,0 +1,103 @@
+require 'bundler/setup'
+
+require 'benchmark/ips'
+require 'json'
+
+require 'rails'
+require 'action_controller/railtie'
+require 'active_record'
+require 'sqlite3'
+require 'ffaker'
+
+TIME = (ENV['BENCHMARK_TIME'] || 5).to_i
```

Comment:
It was 10 firstly, but then I reduced it because waiting wasn't easy. Do you think we will have better results with 10 seconds?